openedx/core/djangoapps/programs/utils.py

```diff
@@ -478,7 +481,8 @@ def get_certificates(user, extended_program):
                 break
 
     program_credentials = get_credentials(user, program_uuid=extended_program['uuid'])
-    if program_credentials:
+    # only include a program certificate if a certificate is available for every course
+    if program_credentials and (len(certificates) == len(extended_program['courses'])):
```

Comment:
@iloveagent57 Yesterday a ticket [LEARNER-2352](https://openedx.atlassian.net/browse/LEARNER-2352) is filed that learners unable to see earned program certificates on program detail pages, this PR went live 3 days ago and may be this check is behind that can you please check?

@schenedx, @bill-filler  FYI.